client/src/utils/formatDate.ts

```diff
@@ -0,0 +1,8 @@
+export const formatDate = (dateString: string): string => {
+    const date = new Date(dateString);
+    const month = date.getMonth() + 1;
+    const day = date.getDate();
+    const dayOfWeek = ["일", "월", "화", "수", "목", "금", "토"][date.getDay()];
```

Comment:
```suggestion
    const DAY_OF_WEEK = ["일", "월", "화", "수", "목", "금", "토"];
    ...
    const today = dayOfWeek[date.getDay()];
```

`["일", "월", "화", "수", "목", "금", "토"]`랑 index가 바로 붙어있으니까 쪼금 헷갈리는 것 같아서 위 예시처럼 따로 상수로 빼주는게 좀 더 가독성이 좋지 않을까 하는 생각입니덩

Reply:
역시!! 너무 좋은생각입니다 :) 리팩토링 여신님 말따르겠습니다!